tests/TestCase/Database/QueryTests/CaseExpressionQueryTest.php

```diff
@@ -277,28 +320,29 @@ public function testOverwrittenReturnType(): void
                         ->setReturnType('boolean'),
```

Comment:
Is this not added to the TypeMap? Is this used or anything?

Reply:
Typed results will only be used in ORM queries, that is what this functional test was meant to cover, same for `testInferredReturnType()`.

Reply:
We should move these tests under the ORM namespace then.

Reply:
I guess so. Maybe we can group them there into some separate file, and not add them to the already humongous ORM query test though?